tests/check/check_h26xsigned_auth.c

```diff
@@ -264,7 +264,7 @@ START_TEST(intact_stream)
   // |settings|; See signed_video_helpers.h.
 
   // Create a list of NALUs given the input string.
-  nalu_list_t *list = create_signed_nalus("IPPIPPI", settings[_i]);
+  nalu_list_t *list = create_signed_nalus("IPPIPPI", settings[_i], false);
```

Comment:
Since you use `create_signed_nalus()` almost exclusively with setting `false`, may I suggest to keep `create_signed_nalus()` as before, which implicitly calls `create_signed_nalus_int()` with `new_priv_key = false`. For more info see comment in `signed_video_helpers.h`